lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,43 @@
+module Overcommit::Hook::PrePush
+  # Replaces the pre-push hook installed by `git lfs install` that is then moved aside by
+  # `overcommit --install`
+  #
+  # Enable this hook in .overcommit.yml like so:
+  #
+  #  PrePush:
+  #    GitLfs:
+  #      enabled: true
+  #      command: ['git', 'lfs', 'pre-push']
+  #
+  # @see https://github.com/github/git-lfs
+  class GitLfs < Base
+    # Overcommit expects you to override this method which will be called
+    # everytime your pre-commit hook is run.
+    def run
+      # Create two arrays to hold our error and warning messages.
+      error_lines = []
+      warning_lines = []
+
+      # Check if git-lfs executable exists
+      path_to_git_lfs = `command -v git-lfs`.strip
+      error_lines << "This repository is configured for Git LFS but 'git-lfs' was not found "\
+                     "on your path. If you no longer wish to use Git LFS, disable GitLfs in "\
+                     ".overcommit.yml." unless File.exist?(path_to_git_lfs)
+
+      result = execute(command, input: pushed_refs)
```

Comment:
`pushed_refs` is an `Array` of `PushedRef` structs. What you want is `input_string`.